packaging/installer/install-required-packages.sh

```diff
@@ -1562,6 +1562,9 @@ validate_tree_centos() {
       fi
     fi
 
+    echo >&2 " > Updating libarchive ..."
```

Comment:
Are we really _updating_ `libarchive` or _installing_ it?

Reply:
libarchive is part of the base instal, but will not update unless you manually ‘install’ it like this.

Reply:
Thanks for the clarification, this answers my question above as well.